language/move-lang/tests/move_check/typing/invalid_type_acquire.exp

```diff
@@ -0,0 +1,481 @@
+error: 
+
+    ┌── tests/move_check/typing/invalid_type_acquire.move:14:9 ───
+    │
+ 14 │         move_to_sender(R { v })
+    │         ^^^^^^^^^^^^^^^^^^^^^^^ Invalid call to move_to_sender.
+    ·
+ 14 │         move_to_sender(R { v })
+    │                        ------- The resource cannot take any type arguments, but found 1 type arguments
+    │
+
+error: 
+
+    ┌── tests/move_check/typing/invalid_type_acquire.move:18:9 ───
+    │
+ 18 │         T,
+    │         ^ Invalid 'acquires' annotation
+    ·
+ 18 │         T,
+    │         - Expected a nominal resource. Found the type: T
```

Comment:
Not needed for this PR, but just a question: What do you think about including the kind of the type in an invalid acquires error setting?

Reply:
A good idea, but another time